pallets/kensetsu/src/lib.rs

```diff
@@ -422,9 +427,13 @@ pub mod pallet {
             borrow_amount: Balance,
         ) -> DispatchResult {
             let who = ensure_signed(origin)?;
-            let interest_coefficient = Self::collateral_infos(collateral_asset_id)
-                .ok_or(Error::<T>::CollateralInfoNotFound)?
-                .interest_coefficient;
+            let collateral_info = Self::collateral_infos(collateral_asset_id)
+                .ok_or(Error::<T>::CollateralInfoNotFound)?;
+            let interest_coefficient = collateral_info.interest_coefficient;
+            ensure!(
+                collateral_amount >= collateral_info.risk_parameters.minimal_collateral_deposit,
+                Error::<T>::CollateralBelowMinimal
+            );
```

Comment:
What if I call `create_cdp(some_sign, XOR, 0, 0)`?
As I see, the code processed `insert_cdp` and `deposit_event` and then doesn't call `deposit_internal` or `borrow_internal`. Is it ok?

Reply:
Yes, it is ok. The user can deposit or borrow later with dedicated extrinsics. This methods combines deposit and borrow for the better UX. The frontend clients don't need to send 3 txs.

Reply:
And what if not? User will not deposit anything later. Just creates e.g. 1000 empty CDPs and go away. Is it still ok?

Reply:
That's why we impose this limit `minimal_collateral_deposit` which is configurable.